src/objects/Application__c.object

```diff
@@ -0,0 +1,268 @@
+                    <default>false</default>
+                    <label>Transfer</label>
+                </value>
+                <value>
+                    <fullName>Graduate</fullName>
+                    <default>false</default>
+                    <label>Graduate</label>
+                </value>
+                <value>
+                    <fullName>Post-Doc</fullName>
+                    <default>false</default>
+                    <label>Post-Doc</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Applying_To__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The academic program or major the applicant is applying to.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The academic program or major the applicant is applying to.</inlineHelpText>
+        <label>Applying To</label>
+        <referenceTo>Account</referenceTo>
+        <relationshipLabel>Applications</relationshipLabel>
+        <relationshipName>Applications</relationshipName>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Preparer__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The Contact preparing the application, often a guardian.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Contact preparing the application. For example, a guardian or a person assisting someone with special needs.</inlineHelpText>
+        <label>Preparer</label>
+        <referenceTo>Contact</referenceTo>
+        <relationshipLabel>Applications (Preparer)</relationshipLabel>
+        <relationshipName>Applications1</relationshipName>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Term__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The term when the applicant wants to enroll.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The term when the applicant wants to enroll.</inlineHelpText>
+        <label>Term</label>
+        <referenceTo>Term__c</referenceTo>
+        <relationshipLabel>Applications</relationshipLabel>
+        <relationshipName>Applications</relationshipName>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <label>Application</label>
+    <listViews>
+        <fullName>All</fullName>
+        <filterScope>Everything</filterScope>
+        <label>All</label>
+    </listViews>
+    <nameField>
+        <displayFormat>App-{0000}</displayFormat>
```

Comment:
App or APP? (Spec shows `APP`)

Reply:
When using prefixes like that I alway go Upper Case, so APP